index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="top-header">
+        <div class="icons-section">
+            <div class="time">
+                <img alt="time icon" src="images/time.png">
+            </div>
+            <div class="group-icons">
+                <img alt="icon of cellural network,wifi and battery" src="images/group.png">
+            </div>
+        </div>
+        <nav class="nav-bar">
+            <a href="#" class="logo-name">John Doe</a>
+           <a href="#" class="menu-icon">
+                <img alt="icon of menu button" src="images/Union.png">
+           </a> 
+        </nav>
+    </header>
+    <main class="main">
+        <section>
+            <article class="headline">
+                <header class="main-header">
+                    <h1>Hey,There I'm Seth</h1> 
+                    <h2>I'am a Software Developer</h2>
+                </header>
+                <p class="headline-desc">
+                    Ican help you build a product, feature or website
+                    through somem of my work and experience! 
+                    if you like what you see and intereseted to work
+                    with me, reach me via:
+                </p>
+                <div class="social-icons">
+                    <a href="#"><img alt="github icon" src="images/git.png" class="git-icon"></a>
+                    <a href="#"><img alt="linkedin icon" src="images/Linkedin.png" class="linkedin-icon"></a>
+                    <a href="#"><img alt="social media icon" src="images/peace.png" class="social-icon"></a>
+                    <a href="#"><img alt="twitter icon" src="images/twitter.png" class="twitter-icon"></a>
+                    <a href="#"><img alt="medium icon" src="images/Vector.png" class="medium-icon"></a>
+                </div>
+            </article>
+        </section>
```

Comment:
Your layout is still breaking on mobile screens as pointed out by the previous reviewer. Please fix it.
Here is a reference image.
![issue](https://user-images.githubusercontent.com/33268927/170524447-51098fe2-2b69-44a9-9356-bead2255c72d.png)